build.psm1

```diff
@@ -2765,13 +2775,20 @@ function Start-CrossGen {
     # the NI PS assemblies, we need to replace the IL PS assemblies with the corresponding NI
     # PS assemblies, but with the same IL assembly names.
     #
-    Write-Verbose "PowerShell Ngen assemblies have been generated. Deploying ..." -Verbose
-    foreach ($assemblyName in $psCoreAssemblyList) {
+    Write-Verbose "PowerShell Ngen assemblies have been generated. Deploying ..."
```

Comment:
You might be right @daxian-dbw . We are already printing another verbose message in which we show the path to the crossgen.exe tool. Alternative, when building, we could use -Verbose, but this prints to much information to the screen which is hard to read and follow. I will revert back to printing the "PowerShell Ngen assemblies have been generated. Deploying ..." message by default.